src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -314,6 +321,225 @@ export default class GroupComparisonStore extends ComparisonStore {
         },
     });
 
+    readonly mutations = remoteData({
+        await: () => [this.samples, this.mutationEnrichmentProfiles],
+        invoke: async () => {
+            if (
+                this.samples.result!.length === 0 ||
+                this.mutationEnrichmentProfiles.result!.length === 0 ||
+                !this.selectedMutationMapperGene
+            ) {
+                return Promise.resolve([]);
+            }
+            const sampleMolecularIdentifiers = getSampleMolecularIdentifiers(
+                this.samples.result!,
+                this.mutationEnrichmentProfiles.result!
+            );
+            const mutations = await client.fetchMutationsInMultipleMolecularProfilesUsingPOST(
+                {
+                    projection: REQUEST_ARG_ENUM.PROJECTION_DETAILED,
+                    mutationMultipleStudyFilter: {
+                        entrezGeneIds: [
+                            this.selectedMutationMapperGene.entrezGeneId,
+                        ],
+                        sampleMolecularIdentifiers,
+                    } as MutationMultipleStudyFilter,
+                }
+            );
+            return mutations;
+        },
+    });
+
+    readonly countProfiledSamples = remoteData({
+        await: () => [this.allSamples, this.coverageInformation],
+        invoke: async () => {
+            let count = 0;
+            if (this.selectedMutationMapperGene) {
+                const sampleKeys = this.allSamples.result!.map(
+                    s => s.uniqueSampleKey
+                );
+                sampleKeys.map(k => {
+                    if (
+                        isSampleProfiled(
+                            k,
+                            this.mutationEnrichmentProfiles.result![0]
+                                .molecularProfileId,
+                            this.selectedMutationMapperGene!.hugoGeneSymbol,
+                            this.coverageInformation.result!
+                        )
+                    ) {
+                        count += 1;
+                    }
+                });
+            }
+            return count;
+        },
+    });
+
+    readonly allSamples = remoteData({
```

Comment:
make sure that we are properly filtering overlapping samples.  (if in fact we want exclude them like other tabs)